Wrapping/Generators/Python/itkExtras.py

```diff
@@ -519,7 +519,7 @@ def imwrite(image_or_filter, filename, compression=False):
     auto_pipeline.current = tmp_auto_pipeline
     writer.Update()
 
-def imread(filename, pixel_type=None, fallback_only=False):
+def imread(filename, pixel_type=None, fallback_only=False, expand_palette=True):
```

Comment:
Since ITK Python only support version 3 now, this argument may best be a [keyword only argument](https://www.python.org/dev/peps/pep-3102/). That is addd `*, ` before this argument.

Reply:
:+1: